views/schedule.mako

```diff
@@ -77,6 +77,7 @@
     </div>
 </div>
 
+<div class="horizontal-scroll">
```

Comment:
Should this not be inside the if since nothing will be inside the div if layout isn't list?